### ISSUE TITLE
Compare to other tools

### ISSUE THREAD

**Issue by elgalu** [otto-de/lineup#5]
Hi, thanks for sharing this.

Could you add to the README how this tool compares to others? or 
What was the motivation to create a new tool that others didn't provide?

##### Alternatives examples
- https://github.com/gemini-testing/gemini
- https://github.com/bfirsh/needle
- https://github.com/cburgmer/csscritic
- https://github.com/rambler-digital-solutions/rspec-visual

There is a longer list here:
https://gist.github.com/cvrebert/adf91e429906a4d746cd

**Comment by StefanDenner** [comment]
Hi Leo!
We did a rewirte in Java: https://github.com/otto-de/jlineup.
We don't use lineup anymore and we will not update it either.
I recommend the Java version, at most in regard of the performance boost compared to this Ruby solution.

At most we implemented this because we wanted to write something on our own ;-)
We wanted a lightweight solution that takes screenshots before a deployment and afterwards, using predefined window sizes for testing responsive web design.
Other tools like gemini for example have a lot of features we do not need, furthermore in lineup we implemented features other tools do not have (or do not have at the first glance), like:

**Several resolutions:** lineup uses a list of resolutions in the config that shall be tested per url/path. I cannot see if for exmaple gemini can do this as well (has only a "windowSize" attribute)
**Cookies**: We needed to configure cookies prior to taking screenshots. lineup can easily set cookies in the browser per config.
**Local Storage**: lineup can also configure local storage key values in the browser, also via the config.

I hope this helps.
Best regards,
Stefan
